OpenEXR/IlmImf/ImfDeepScanLineInputFile.cpp

```diff
@@ -1839,7 +1839,7 @@ readSampleCountForLineBlock(InputStreamMutex* streamData,
     //
 
 
-    if (sampleCountTableDataSize < data->maxSampleCountTableSize)
+    if (sampleCountTableDataSize < Int64(data->maxSampleCountTableSize))
```

Comment:
inconsistent initialization vs our preferred static_cast. Here and a couple other places...

Reply:
I've converted all constructor casts to static_cast